gensim/models/word2vec.py

```diff
@@ -309,6 +310,8 @@ def __init__(self, sentences=None, size=100, alpha=0.025, window=5, min_count=5,
         self.hashfxn = hashfxn
         self.iter = iter
         if sentences is not None:
+            if isinstance(sentences, GeneratorType):
+                raise TypeError("You can't pass a generator as the sentences argument. Try an iterator.")
```

Comment:
There's a typo here that slipped the code review -- "iterator" should read "iterable". Can you fix this?